app/views/course/assessment/submission/submissions/reload_answer.js.erb

```diff
@@ -3,7 +3,7 @@ $('<%= "#answer_#{@answer.id}"%>').replaceWith(
 );
 
 <% if @assessment.autograded? && next_unanswered_question.nil? %>
-  $('input.finalise').show();
+  $('input.finalise').first().show();
```

Comment:
Cause there are 2 finalise buttons rendered, one on top, one below, so only show the top one. 

At some point I will probably refactor some of the views, or just wait for it to be replaced by react.